cmd/operator/deploy/operator/05-deployment.yaml

```diff
@@ -18,14 +18,14 @@ metadata:
   namespace: gmp-system
   name: gmp-operator
   labels:
+    app: managed_prometheus_operator
```

Comment:
It looks like you moved this without reformatting to dashes.

Reply:
done